### ISSUE TITLE
Need BundleTransformer.NUglify new package release

### ISSUE THREAD

**Issue by jzabroski** [Taritsyn/BundleTransformer#54]
See my comments here: https://github.com/Taritsyn/BundleTransformer/commit/beb83bd62bc0ad5150f6775ed10efc7b5fc87890#r67181806

**Comment by Taritsyn** [comment]
Hello, John!

This problem has been resolved in [version 1.12.46](https://github.com/Taritsyn/BundleTransformer/releases/tag/v1.12.46).

Basically, you could use the following command to update the package:

```
nuget update "PATH-TO-CSPROJ-FILE" -Id NUglify -Version 1.17.8
```

In case of an error “Could not load file or assembly 'NUglify, Version=1.17.3.0, Culture=neutral, PublicKeyToken=15bc7810aec21b5e' or one of its dependencies. …”, add the following setting to the `Web.config` file:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  …
  <runtime>
    …
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      …
      <dependentAssembly>
        <assemblyIdentity name="NUglify" publicKeyToken="15bc7810aec21b5e" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-1.17.8.0" newVersion="1.17.8.0" />
      </dependentAssembly>
      …
    </assemblyBinding>
    …
  </runtime>
  …
</configuration>
```

**Comment by jzabroski** [comment]
@Taritsyn Agree, it is resolved.  Closing this.  TIL that Nuget chooses the lowest applicable version for transitive references, in order to, for example, avoid upgrading someone into a zero day exploit.
